RELEASING.md

```diff
@@ -47,7 +47,9 @@ Next, update the
 
 Create a PR to mark the new release in README.md and CHANGELOG.md on the main branch.
 
-Finally, update the files `website_docs` directory to point at the newly released version.
+Finally, update the the [website docs][] to refer to the newly released version.
```

Comment:
```suggestion
Finally, update the the [website docs] to refer to the newly released version.
```

Reply:
By using a trailing `[]` on markdown link references, editors like VS Code give you visual feedback if your link is defined or not. So I'd suggest keeping it, but it's your call.

Reply:
(This is _really_ useful as-you-type feedback. :))

Reply:
does it makes sense to add it above also on the `[Manual instrumentation]` link?

(also just noticed an extra `the`)

```suggestion
Finally, update the [website docs][] to refer to the newly released version.
```

Reply:
Thanks for catching the stutter.

Reply:
> does it makes sense to add it above also on the `[Manual instrumentation]` link?

Done